catalog/core/catalog-core-sourcemetricsplugin/src/main/java/ddf/catalog/metrics/source/SourceMetricsImpl.java

```diff
@@ -546,7 +543,6 @@ public SourceMetric(Metric metric, String sourceId, RrdJmxCollector collector) {
         public SourceMetric(Metric metric, String sourceId, RrdJmxCollector collector,
```

Comment:
Good catch the sourceId really was not used at all. It is actually a key in a map rather than being used in this class. Probably missed in the original design of metrics. !fixed